### PR TITLE
Only call getPerms on readable files to ensure we avoid a fatal error

### PR DESCRIPTION
**Pull request by willmot**
Bunch of coding standards fixes

### CODE REVIEW

File: admin/schedule-form-excludes.php [hunk 1]
```diff
@@ -303,24 +291,21 @@ class="dashicons dashicons-update"
 					</td>
 
 					<td>
-						<?php echo esc_html( substr( sprintf( '%o', $file->getPerms() ), - 4 ) ); ?>
+						<?php if ( ! $is_unreadable ) {
```
Comment (willmot) [inline admin/schedule-form-excludes.php]: This is the line which matters, we needed to check if the file was readable before calling `getPerms`

File: admin/schedule-form-excludes.php [hunk 2]
```diff
@@ -103,102 +102,98 @@
...
+				</th>
 
-					$root = new \SplFileInfo( Path::get_root() );
+				<td class="column-filesize">
 
-					$size = $site_size->filesize( $root );
+					<?php if ( Site_Size::is_site_size_being_calculated() ) { ?>
 
-					if ( false !== $size ) {
+						<span class="spinner is-active"></span>
```
Comment (pdewouters) [inline admin/schedule-form-excludes.php]: missing opening bracket?

File: admin/schedule-form-excludes.php (hunk 1 above)
Comment (pdewouters) [inline admin/schedule-form-excludes.php]: where is the var defined?

Comment (willmot) [inline admin/schedule-form-excludes.php]: Just here: https://github.com/humanmade/backupwordpress/blob/fatal-get-perms/admin/schedule-form-excludes.php#L215

**Pull request merged**